samples/bluetooth/le_periph_past/src/main.c

```diff
@@ -235,17 +235,28 @@ static void on_appearance_get(uint8_t conidx, uint32_t metainfo, uint16_t token)
 	}
 }
 
-static void on_ctrl_hw_error(enum co_error hw_err_code)
+#if !CONFIG_ALIF_BLE_ROM_IMAGE_V1_0 /* ROM version > 1.0 */
+static void on_gapm_err(uint32_t metainfo, uint8_t code)
 {
-	LOG_ERR("hw_err_code: %u", hw_err_code);
+	LOG_ERR("gapm error %d", code);
 }
+#else
+static void on_gapm_err(enum co_error err)
+{
+	LOG_ERR("gapm error %d", err);
+}
+#endif /* !CONFIG_ALIF_BLE_ROM_IMAGE_V1_0 */
 
 static uint16_t utils_create_adv(void)
 {
 	static const gapm_le_adv_create_param_t adv_create_params = {
 		.prop = GAPM_ADV_PROP_UNDIR_CONN_MASK,
 		.disc_mode = GAPM_ADV_MODE_GEN_DISC,
+#if !CONFIG_ALIF_BLE_ROM_IMAGE_V1_0 /* ROM version > 1.0 */
```

Comment:
if we create static do we need to iit zero parameters? @VeijoPesonen It will make this then even simplifier. Now 1.2 rom have only different name at structure but you init value to 0.